ParserXML/Parser.cs

```diff
@@ -1,46 +1,74 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Linq;
+using System.Runtime.Remoting.Services;
 using System.Xml.Linq;
 
-public class Contact
-{
-    public string Name { get; set; }
-    public Dictionary<string, string> Phones { get; set; }
-    public Address Address { get; set; }
-    public decimal? NetWorth { get; set; }
-
-    public Contact()
-    {
-        Phones = new Dictionary<string, string>();
-    }
-}
-
-public class Address
-{
-    public string Street1 { get; set; }
-    public string City { get; set; }
-    public string State { get; set; }
-    public string Postal { get; set; }
-}
-
 public class Program
 {
+    /// <summary>
+    /// New list
+    /// </summary>
     public static List<Contact> Contacts = new List<Contact>();
 
-    public static void Main(string[] args)
+    public static void Main()
     {
-        string filePath = "contacts.xml"; 
+        string filePath = "contacts.xml";
         ParseContacts(filePath);
 
         while (true)
         {
-            Console.WriteLine("\n1. Display All Contacts");
+            Console.WriteLine("1. Display Contacts");
+            Console.WriteLine("2. Search Contacts");
+            Console.WriteLine("3. Exit");
             string option = Console.ReadLine();
 
+            switch (option)
+            {
+                case "1":
+                    DisplayContacts();
+                    break;
+                case "2":
+                    SearchContacts();
+                    break;
+                case "3":
+                    return;
+                default:
+                    Console.WriteLine("Invalid option");
+                    break;
+            }
         }
     }
+    /// <summary>
+    /// Class for properties in Contact
+    /// </summary>
+    public class Contact
+    {
+        public string Name { get; set; }
+        public Dictionary<string, string> Phones { get; set; }
+        public Address Address { get; set; }
+        public decimal NetWorth { get; set; }
 
+        public Contact()
+        {
+            Phones = new Dictionary<string, string>();
+        }
+    }
+    /// <summary>
+    /// Class for properties in Address
+    /// </summary>
+    public class Address
+    {
+        public string Street { get; set; }
+        public string City { get; set; }
+        public string State { get; set; }
+        public string Postal { get; set; }
+    }
+    /// <summary>
+    /// Parsing contacts
+    /// </summary>
+    /// <param name="filePath">The parameter that points to the path to the file</param>
```

Comment:
Don't be cap. We know that it is parameter. You can write it like: Path to the .xml file